virt/rhevm/rhevm.py

```diff
@@ -161,12 +165,9 @@ def getHostGuestMapping(self):
                     guest_id)
                 state = virt.Guest.STATE_UNKNOWN
 
-            mapping[host_id].append(virt.Guest(guest_id, self, state))
+            hosts[host_id].guestIds.append(virt.Guest(guest_id, self, state))
 
-        mapping_with_ids = {}
-        for host_id, vms in mapping.items():
-            mapping_with_ids[host_ids[host_id]] = vms
-        return mapping_with_ids
+        return {'hypervisors': [h for id, h in hosts.items]}
```

Comment:
This raises an error (missing function call of the `.items`), it's this what you meant?
```python
return {'hypervisors': hosts.values()}
```